src/maps/index.js

```diff
@@ -83,3 +83,33 @@ nameInfos.forEach((d) => {
 export function getInfoByName(name) {
   return infoLookup.get(String(name).toLowerCase());
 }
+
+/**
+ * computes the population of the mega county as state - defined county populations
+ * @param {NameInfo} megaCounty
+ * @param {Map<string, any>} data
+ */
+export function computeMegaCountyPopulation(megaCounty, data) {
+  if (!megaCounty || !data || megaCounty.level !== levelMegaCountyId) {
+    return null;
+  }
+  const state = getInfoByName(megaCounty.postal);
+  if (!state || state.population == null || Number.isNaN(state.population)) {
+    return null;
+  }
+  let base = state.population;
+  data.forEach((_, key) => {
```

Comment:
Style nit (optional fix).  You can use a reduce instead of a forEach which would allow you use immutable variables.  E.g.
```
let definedCountyPop = allCounties.map(/*maps county to population*/).reduce((acc, cur)=> { return acc + cur})
let restOfState = state.population - definedCountyPop
```